workerapi/server/server.go

```diff
@@ -144,9 +144,9 @@ func (h *handler) QueryWorker() (*worker.WorkerStatus, error) {
 
 // Implements worker.thrift Worker.Run interface
 func (h *handler) Run(cmd *worker.RunCommand) (*worker.RunStatus, error) {
-	defer h.stat.Latency(stats.WorkerServerRunLatency_ms).Time().Stop() // TODO errata metric - remove if unused
+	defer h.stat.Latency(stats.WorkerServerStartRunLatency_ms).Time().Stop()
```

Comment:
This was already there - it basically reports how long it takes to get a run id (start the task).  I renamed it to include 'Start' to (try to) make that more clear.